### PR TITLE
Updates to pipeline

### PR DESCRIPTION
**Pull request by poornimaramesh**
Reviewer: @ziakhan04 

Estimate: ~5mins

---
### Goal
Update pipeline code with all the latest changes

### Changes
Updated the pipeline code with the translation layer, top-k values search, etc.

## How has this been tested?
I re-ran the demo notebook and experiment notebooks. Things seem to work.

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the requirements (if applicable)
- [x] I have updated affected documentation (if applicable)

### CODE REVIEW

File: askametric/query_processor/tools.py [hunk 1]
```diff
@@ -146,27 +147,30 @@ async def get_common_column_values(
...
-                        GROUP BY {column}
-                        ORDER BY COUNT(*) DESC
-                        LIMIT {num_common_values};
-                        """
-                    )
-                )
+                query = f"""
+                SELECT DISTINCT({column})
+                FROM {table}
+                GROUP BY {column}
```
Comment (ziakhan04) [inline askametric/query_processor/tools.py:166]: This breaks the EG pipeline. Look at [this ticket](https://idinsight.atlassian.net/browse/LIAM-171)

**Review by ziakhan04 — CHANGES_REQUESTED** [review]
Please resolve the get best columns query function. It breaks the EG pipeline

File: askametric/query_processor/tools.py (hunk 1 above)
Comment (poornimaramesh) [reply askametric/query_processor/tools.py:166]: fixed

**Review by ziakhan04 — APPROVED** [review]
Works! Thanks!

**Pull request merged**